superset/connectors/base/models.py

```diff
@@ -60,8 +60,8 @@ class BaseDatasource(AuditMixinNullable, ImportMixin):
     perm = Column(String(1000))
 
     sql = None
-    owners = None
```

Comment:
What's the implication of removing this default value?

Reply:
I don't think `owners` and `update_from_object_fields` are ever referenced before they are assigned, so the default value isn't really necessary. But tbh, I just removed the default value so I could add a non optional type, so I'm fine with putting it back in.